src/medCore/gui/toolboxes/medAbstractSelectableToolBox.h

```diff
@@ -35,13 +35,13 @@ class MEDCORE_EXPORT medAbstractSelectableToolBox : public medToolBox
 
     void setSelectorToolBox(medSelectorToolBox *toolbox);
 
+    //! Called when scrolled in medSelectorToolBox
+    virtual void changeSelectedToolBoxEvent(){}
```

Comment:
I'm ok to use "clear()", i thought it would be more understandable if we used a "changeSelectedToolBoxEvent" function (or something like that), but i understand.